kyaml/fn/runtime/runtimeutil/functiontypes.go

```diff
@@ -33,6 +33,26 @@ const (
 	UserNobody ContainerUser = "nobody"
 )
 
+// ContainerNetworkName is a type for network name used in container
+type ContainerNetworkName string
```

Comment:
It's sufficient to just define the type, to document how it flows through the code and have type checks keep it distinct from bare string usage.  It's premature to add String(), IsEmpty(), Set() etc.  The type still behaves like a string for the most part.